src/api/FileVersionAPI.js

```diff
@@ -77,11 +77,20 @@ class FileVersionAPI extends API {
      * @param {Params} queryParams - Key-value map of querystring params
      * @return {void}
      */
-    successHandler = (data: Data, queryParams: Params): void => {
+    successHandler = (data: Data, queryParams: Params): Promise<any> => {
+        if (data.type === 'error' || !Array.isArray(data.entries)) {
```

Comment:
Is the response 200 when in this case? If not, can it go into an error callback?

Reply:
Went WAY back but don't really have a solid reasoning behind why this is there :-/